### PR TITLE
docs: update newcomer guide

### LINKED ISSUE #3926: docs: Mention that the only way to familiarise yourself with the codebase is to practice and take up new issues
Opened via [gitter](https://gitter.im/coala/coala/?at=58c9581efe6a638b1ac927d6) by @adtac

### PR DESCRIPTION
**Pull request by sinwar**
Mention that the only way to familiarise yourself with the codebase is to practice and take up new issues.

Fixes #3926

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by NiklasMM** [comment]
I know, it has been dictated by the issue, but I don't like the phrasing "the only way". Because it's really not true :smile: 

I would rephrase it as "the best way". What do you think?

**Comment by NiklasMM** [comment]
Also I'd put it somewhere under "Step 3 Picking up an issue"

> Now it is time to pick an issue. It is the best way to familiarise yourself with the codebase. Here is the link that will lead you to Newcomers issues.

or something like that.

**Comment by sinwar** [comment]
Yup it's the best way :D.

**Comment by NiklasMM** [comment]
Looks good! Now [rebase](http://www.coala.io/rebase) your branch and we're good to merge.

**Comment by NiklasMM** [comment]
ack 519a35f

**Comment by sinwar** [comment]
did @NiklasMM

**Comment by NiklasMM** [comment]
reack f67221c

**Comment by NiklasMM** [comment]
@rultor merge
